### ISSUE TITLE
Change Email, send PM again

### ISSUE THREAD

**Issue by wads24** [bb3mobi/apwa#18]
If a user changes their email address and board required reactivation they are send a second welcome PM when they click on activation link for changing email

**Comment by ProjektPremium** [comment]
Yes, they get 2 PMs after they activate. Is there a way to fix this?
